app/src/main/kotlin/com/thundermaps/apilib/android/api/resources/TracedContacts.kt

```diff
@@ -0,0 +1,31 @@
+package com.thundermaps.apilib.android.api.resources
+
+import com.google.gson.annotations.Expose
+data class TracedContacts(
+    val traced_contacts: List<TracedContact> = ArrayList()
+) : SaferMeDatum {
+    override fun toString(): String {
+        val sb = StringBuilder()
+        sb.append("tracedContacts $traced_contacts")
+        return sb.toString()
+    }
+}
+
+data class TracedContact(
+    @Expose
+    val timestamp: String? = null,
+
+    @Expose
+    val trace_type: String = "bluetooth",
```

Comment:
Could you use an enum for this? If not I'm wondering if it would make sense to use the same bluetooth API key that is used for the API models? e.g. "automatic_from_client_bluetooth"

Reply:
basically this will come from obsidian service. is just a default in the case it is not sent. in obsidian service, it comes as automatic_from_client_bluetooth for ble. but I assume that in a  midterm we will be sending other data also